model-optimizer/extensions/ops/non_max_suppression.py

```diff
@@ -57,31 +56,53 @@ def backend_attrs(self):
 
     @staticmethod
     def infer(node: Node):
+        num_of_inputs = len(node.in_ports())
+        assert 2 <= num_of_inputs <= 6, \
```

Comment:
6 inputs can be in the opset5 version of the operation only. Other version of the operation must have less.

Reply:
Fixed.